portal/models/telecom.py

```diff
@@ -12,37 +12,49 @@ class Telecom(object):
     class wraps common functions.
 
     """
-    def __init__(self, phone=None, email=None, fax=None):
+    def __init__(self, phone=None, email=None, fax=None, alt_phone=None):
         self.phone = phone
         self.email = email
         self.fax = fax
+        self.alt_phone = alt_phone
 
     def __str__(self):
-        return "Telecom: {0.phone} {0.email}".format(self)
+        return "Telecom: {0.phone} {0.alt_phone} {0.email}".format(self)
 
     @classmethod
     def from_fhir(cls, data):
         telecom = cls()
         for item in data:
             attr = item.get('system')
             value = item.get('value')
+            use = item.get('use')
```

Comment:
It should already be setting to default if they don't pass one. But I agree, we should throw a warning if they send an invalid one, I'll add that in